### PR TITLE
Lowers Chance of Shadowling and Darkspawn

### PR DESCRIPTION
**Pull request by JamieD1**
Noone likes them

:cl:  
tweak: Lowers Darkspawn chance and reenabled shadowling as lower chance
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
Missing rationale

**Comment by JamieD1** [comment]
> Missing rationale

Its shit.

**Comment by ynot01** [comment]
> Its shit.

Million times better than slings

**Comment by JamieD1** [comment]
> > Its shit.
> 
> Million times better than slings

So still a million times shit.

**Comment by ThatLing** [comment]
> > Missing rationale
> 
> Its shit.

How is it shit?

**Comment by JamieD1** [comment]
> > > Missing rationale
> > 
> > 
> > Its shit.
> 
> How is it shit?

Noone likes it. Its unfun, boring, stupid, ends too quickly. Literally look at every other fuckers opinion.

File: config/game_options.txt [hunk 1]
```diff
@@ -126,7 +126,7 @@ PROBABILITY CULT 5
 PROBABILITY CLOCKWORK_CULT 5
 
 # Snowflakes
-PROBABILITY SHADOWLING 0 //For now
+PROBABILITY SHADOWLING 2 //For now
```
Comment (SomeguyManperson) [inline config/game_options.txt]:
```suggestion
PROBABILITY SHADOWLING 0
```

**Comment by Hislyna** [comment]
Can we make both them chances 0?

**Comment by MajManatee** [comment]
both are terrible :)

File: config/game_options.txt [hunk 2]
```diff
@@ -126,7 +126,7 @@ PROBABILITY CULT 5
 PROBABILITY CLOCKWORK_CULT 5
 
 # Snowflakes
-PROBABILITY SHADOWLING 6
+PROBABILITY SHADOWLING 2
```
Comment (JamieD1) [inline config/game_options.txt]:
```suggestion
PROBABILITY SHADOWLING 2
```

**Pull request closed without merging**